PProPanel/jsx/Premiere.jsx

```diff
@@ -248,19 +236,18 @@ $._ext_PPRO={
         
         if (app.project != null) {
 
-        // Find or create a target bin.
-
+	        // If panel can't find the target bin, it creates it.
         var nameToFind = 'Targeted by PProPanel import';
```

Comment:
Argh ignore me. My fault! I thought the code was part of function searchForBinName and I could not understand why you were not reassigning nameToFind. I did not see the importFiles function definition.